PROCESS.md

```diff
@@ -0,0 +1,279 @@
+# Haskell Security Response Team Process
+
+> Copyright 2017, OpenStack Vulnerability Management Team
+> Copyright 2023, Haskell Security Response Team
+> This work is licensed under a Creative Commons Attribution 3.0 Unported License.
+>   http://creativecommons.org/licenses/by/3.0/legalcode
+> Original version: https://opendev.org/openstack/ossa/src/branch/master/doc/source/vmt-process.rst
+
+The Haskell Security Response Team (HSRT) is responsible for coordinating
+the progressive disclosure of a vulnerability.
+
+Members of the team are independent and security-minded folks who ensure
+that vulnerabilities are dealt with in a timely manner and that
+downstream stakeholders are notified in a coordinated and fair manner.
+Where a member of the team is employed by a downstream stakeholder, the
+member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the disclosure of vulnerability in the early stages,
+membership of this team is intentionally limited to a small number of
+people.
+
+## Supported versions
+
+The Vulnerability Management team coordinates patches fixing
+vulnerabilities in packages published via Hackage.
+
+## Process
+
+Each security bug is assigned a HSRT *coordinator* (member from the
+haskell security response team) that will drive the fixing and
+disclosure process. Here are the steps we follow.
+
+### Reception
+
+Private report can be received by email to [security-advisories@haskell.org](mailto:security-advisories@haskell.org),
+or via a [private issue on the security-advisories repo](https://github.com/haskell/security-advisories/security/advisories/new).
+
+Public report can be submitted as a [regular issue on the security-advisories repo](https://github.com/haskell/security-advisories/issues).
+
+The first steps performed by the HSRT are to:
+
+-   create a bug if one does not yet exist.
+-   check that the report indicates the correct project and adjust as
+    necessary.
+-   contact the project's maintainer for confirmation
+    of impact and determination of affected version.
+
+### Patch development
+
+For a private report, the reporter (automatic if reported directly as a
+bug) and the affected projects' maintainers plus anyone they deem necessary
+to develop and validate a fix are added to the bug's notified list.
```

Comment:
What is a notified list?

Reply:
That would be the issue notification/participant list, but I'm removing this part since it does not apply for us.